docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -55,6 +55,26 @@ There you can find links to upgrade notes for other versions too.
         ```
 - update your tests to use interfaces of factories fetched from dependency injection container
     -  update tests same way as in PR ([#970](https://github.com/shopsys/shopsys/pull/970/files))
+- check your VAT calculations after it was modified in `shopsys/framework` ([#1129](https://github.com/shopsys/shopsys/pull/1129))
+    - see [the description of the PR](https://github.com/shopsys/shopsys/pull/1129) to understand the scope of this change
+    - run DB migrations on your demo data and on a copy of production database (if possible) to see that it works with your data well
```

Comment:
It sounds like we are not sure if it works. Please rephrase. 

I would recommend starting this paragraph with "if you modified the price calculation or you altered the prices in the database directly"